test/integration/pr_test.go

```diff
@@ -51,6 +56,46 @@ func TestPRGithubDecorationSuccessCase(t *testing.T) {
 	assert.NilError(t, err, "Error should be nil")
 }
 
+func TestPRGithubOnPremDecorationSuccessCase(t *testing.T) {
```

Comment:
what are you checking here?
try to uw please when_given_then(should)

Reply:
TestPRGithubDecoration_WhenUseCodeRepositoryFlag_ShouldSuccess